internal/lib/constants.go

```diff
@@ -99,11 +99,11 @@ const (
 	GatewayFinalizer                           = "gateway.ako.vmware.com"
 	ClusterStatusCacheKey                      = "cluster-runtime"
 	AviObjDeletionTime                         = 30 // Minutes
-	AKOConditionType                           = "akoStatus"
+	AKOConditionType                           = "ako.vmware.com/ObjectDeletionInProgress"
 	AKOStatefulSet                             = "ako"
-	ObjectDeletionStartStatus                  = "objDeletionStarted"
-	ObjectDeletionDoneStatus                   = "objDeletionDone"
-	ObjectDeletionTimeoutStatus                = "objDeletionTimeout"
+	ObjectDeletionStartStatus                  = "ObjectDeletionStarted"
+	ObjectDeletionDoneStatus                   = "ObjectDeletionDone"
+	ObjectDeletionTimeoutStatus                = "ObjectDeletionTimeout"
```

Comment:
since ObjectDeletion is common in all enums, can we chuck it out to just be `Started/Done/Timedout`?

Reply:
Sounds fine to me. @sudswasavi ?

Reply:
yeah just on that thought, I wonder how we signify what objects these are. In this case, there are LB Objects I think. Thoughts?